scripts/test.sh

```diff
@@ -1,4 +1,4 @@
 set -e
 # Run linter
-find . -type f -name "*.js" -not -path "*node_modules*" -not -path "*dataconnect-sdk/*" \
-  | xargs eslint
+find . -type f -name "*.js" -not -path "*node_modules*" -not -path "*dataconnect-sdk/*" -not -path "*ai/sample-app/*" \
```

Comment:
I think you changed the dir name at some point after this, needs to be `ai-react-app` now